### PR TITLE
build(deps): Adapt to changes in go-utils

### PR DESCRIPTION
**Pull request by bacherfl**
Integration test run: https://github.com/keptn/keptn/actions/runs/2574776102

### CODE REVIEW

File: configuration-service/go.mod [hunk 1]
```diff
@@ -11,15 +11,15 @@ require (
...
+	github.com/keptn/go-utils v0.16.1-0.20220628065837-3ed2fc6cf1ed
 	github.com/mholt/archiver/v3 v3.5.1
 	github.com/otiai10/copy v1.7.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.1
 	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/apimachinery v0.22.11
-	k8s.io/client-go v0.22.11
+	k8s.io/apimachinery v0.24.2
```
Comment (odubajDT) [inline configuration-service/go.mod:21]: can you please stay at version 22.11 ?

File: test/go-tests/go.mod [hunk 2]
```diff
@@ -6,51 +6,59 @@ require (
...
 	github.com/google/uuid v1.3.0
 	github.com/imroc/req v0.3.2
-	github.com/keptn/go-utils v0.16.1-0.20220627120041-8d145bc90294
+	github.com/keptn/go-utils v0.16.1-0.20220628065837-3ed2fc6cf1ed
 	github.com/mholt/archiver/v3 v3.5.1
 	github.com/stretchr/testify v1.7.1
-	k8s.io/api v0.22.11
-	k8s.io/apimachinery v0.22.11
-	k8s.io/client-go v0.22.11
+	k8s.io/api v0.24.2
```
Comment (odubajDT) [inline test/go-tests/go.mod:12]: here as well

Comment (bacherfl) [reply test/go-tests/go.mod]: done

**Comment by odubajDT** [comment]
pls update also renovate https://github.com/keptn/keptn/blob/master/.github/renovate.json#L17

**Review by odubajDT — APPROVED** [review]

**Pull request merged**